### ISSUE TITLE
Nested fields validation

### ISSUE THREAD

**Issue by PierreGUI** [DavyJonesLocker/client_side_validations#705]
Hello again,
I have an issue where validation for nested models is not generated. Also, adding validation on the input itself (wrapped with `field_for`) won't have any effect.

### Versions
`client_side_validations 9.3.1`
`client_side_validations-simple_form 6.2.0`
`Rails 5.0.1`

### Model
Checkout `has_many :orders`, order `belongs_to :checkout` and `validates :id, presence: true, allow_blank: false`

### Form
I use simple_form but I've made sure it doesn't work without either:
```
<%= co.fields_for :order, validate: true do |o| %>
    <%= o.input :id, label: t("label.order_id"), validate: { presence: true, allow_blank: false } %>
  <% end %>
```

### Example project
I used same as before (for previous issue), where I added nested model / fields (checkout has many orders):
https://github.com/PierreGUI/CSV-isValid

### Actual
No validation happens for `order: id`. Nor from model or custom validation in form tags.

### Expected
Order's id is validated against `validates` from `order.rb`.
Or, order's id is validated against `validate: { presence: true, allow_blank: false }` on form's field.

What am I doing wrong?
Thanks for your help.

**Comment by tagliala** [comment]
Same as https://github.com/DavyJonesLocker/client_side_validations/issues/680#issuecomment-259221769 ?

**Comment by PierreGUI** [comment]
Might be same same. I can indeed get the validation working when I instantiate the Order object 'manually' in my test project (which is not exactly what I have in prod).
Problem is the validation happens in a form linked to [Devise](https://github.com/plataformatec/devise) so I would have to include it in test project (eventually later).

So far I'm going to change my form to not use `fields_for`, because I also have an `attr_accessor` field I can't validate with CSV.

I'm curious to know how attaching validators to field works though, doesn't seems to behave as I expect.
Again, thanks for your time.

**Comment by tagliala** [comment]
>  I also have an attr_accessor field I can't validate with CSV.

I'm using CSV to validate some `attr_accessor` fields, e.g.:

```rb
class Contact < ActiveRecord::Base
  attr_accessor :privacy

  # ...

  validates :privacy, acceptance: true, on: :create
end
```

You mean that you need some server side validations?

**Comment by tagliala** [comment]
This will be fixed in the upcoming 9.3.2
